src/main/java/com/qaprosoft/carina/demo/gui/pages/ArticlePage.java

```diff
@@ -0,0 +1,43 @@
+package com.qaprosoft.carina.demo.gui.pages;
+
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.ExtendedWebElement;
+import com.qaprosoft.carina.core.gui.AbstractPage;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+public class ArticlePage extends AbstractPage {
+
+    @FindBy(xpath = "//h1[@class='article-info-name']")
+    private ExtendedWebElement title;
+
+    @FindBy(xpath = "//h3[text()='RedMagic Watch now available globally']")
```

Comment:
bad element
this element only for one special article
what will happen if you open another article